taskcluster/docker/decision/Dockerfile

```diff
@@ -21,9 +15,5 @@ ADD topsrcdir/requirements/base.txt /setup/requirements.txt
 ADD system-setup.sh /setup/system-setup.sh
 RUN bash /setup/system-setup.sh
```

Comment:
Isn't this already run in the parent image?

Reply:
The two files are different now, the parent one sets up run-task and this one sets up the rest of Taskgraph